tests/fixtures/tricky_extra_files/app.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+
+from flask import Flask
+
+APP_DIR = Path(__file__).parent
+
+app = Flask(__name__)
+
+
+freeze_config = {
+    'extra_files': {'static': {'copy_from': str(APP_DIR / 'static_dir')}},
```

Comment:
Does you make sense to add configuration as:
```python
'/static2/': {'copy_from': str(APP_DIR / 'static_dir2')}
```
to check if we handle slashes correctly?
I have prepared code for that if you'd like add this case...